terraform/terraform_spec.rb

```diff
@@ -18,7 +18,7 @@
   it "has the expected Terraform version" do
     expect(
       command("terraform version").stdout
-    ).to include("Terraform v0.7.3")
+    ).to include("Terraform v0.7.5")
```

Comment:
Well, I thing we should check for the as explicit version as possible. Otherwise we can test if we use `Terraform v0` and it will always pass.